sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -404,6 +406,14 @@ export class AssessmentController {
           saveAssessmentCalculationStartDate,
           getProgramYearTotalAwards,
         ]);
+        // Fetch the latest CSLP balance for the student.
+        const studentLoanBalance =
+          await this.studentLoanBalanceSharedService.getLatestCSLPBalance(
+            studentId,
+          );
+        result["latestCSLPBalance"] = studentLoanBalance[0]
```

Comment:
The property `latestCSLPBalance` is always be present in the return and can be explicitly declared as `isReadyForCalculation`, right? Please let me know if I am missing something.

Reply:
Missed it, changed now